MAUI/Expander/Getting-Started.md

```diff
@@ -0,0 +1,400 @@
+                                <Label CharacterSpacing="0.25" FontFamily="Roboto-Regular" FontSize="14" Text="Address" Grid.Column="1" VerticalOptions="CenterAndExpand"/>
+                            </Grid>
+                        </syncfusion:SfExpander.Header>
+                        <syncfusion:SfExpander.Content>
+                            <Grid Padding="18,8,18,18" Grid.Row="1" RowSpacing="1">
+                                <Grid.Resources>
+                                    <Style TargetType="Label">
+                                        <Setter Property="FontFamily" Value="Roboto-Regular"/>
+                                    </Style>
+                                </Grid.Resources>
+                                <Grid.RowDefinitions >
+                                    <RowDefinition Height="Auto"/>
+                                    <RowDefinition Height="Auto"/>
+                                    <RowDefinition Height="Auto"/>
+                                </Grid.RowDefinitions>
+                                <Label CharacterSpacing="0.25"  FontSize="14" Text="Alex" FontAttributes="Bold"/>
+                                <Label CharacterSpacing="0.25"  FontSize="14" Text="No.8 Blossom St, Washington, DC 20019" Grid.Row="1"/>
+                                <Label CharacterSpacing="0.25"  FontSize="14" Text="(202) 547-3555" Grid.Row="2"/>
+                            </Grid>
+                        </syncfusion:SfExpander.Content>
+                    </syncfusion:SfExpander>
+                </Border>
+            </StackLayout>
+        </ScrollView>
+    </ContentPage.Content>
+</ContentPage>
+
+Now, run the application to render the desired output.
+
+![.Net MAUI Expander](Images/maui-expander-with-gettingstarted.png)
+
+
+## Animation duration
+
+The `SfExpander` allows you to customize the expanding and collapsing duration by using the `AnimationDuration` property. By default, the animation duration is `200 milliseconds`. 
+
+{% tabs %}
+{% highlight xaml %}
+    <syncfusion:SfExpander x:Name="expander" AnimationDuration="250"/>
+{% endhighlight %}
+{% highlight c# %}
+    expander.AnimationDuration = 250;
+{% endhighlight %}
+{% endtabs %}
+
+## Animation easing
+
+The `SfExpander` allows you to customize the rate of change of parameter over time or animation style by using the `AnimationEasing` property. By default, the animation easing is `Linear`. 
+
+{% tabs %}
+{% highlight xaml %}
+         <syncfusion:SfExpander x:Name="expander" AnimationEasing="SinOut"/>       
+{% endhighlight %}
+{% highlight c# %}
+    expander.AnimationEasing = Syncfusion.Maui.Expander.AnimationEasing.SinOut;
+{% endhighlight %}
+{% endtabs %}
+
+## Expand and Collapse 
+
+The `SfExpander` allows you to programmatically expand and collapse by using the `IsExpanded` property of SfExpander. The user can control the expand and collapse interactions by handling the `Expanding` and `Collapsing` events.
```

Comment:
The `SfExpander` allows you to programmatically expand and collapse by using the `IsExpanded` property of the SfExpander. The user can control the expand and collapse interactions by handling the `Expanding` and `Collapsing` events.